services/dewey/src/dewey/core.clj

```diff
@@ -32,72 +33,55 @@
       (do
         (log/info "Failed to find elasticsearch. Retrying...")
         (Thread/sleep 1000)
-        (recur props)))))
+        (recur)))))
 
 
 (defn- init-irods
-  [props]
-  (irods/init (get props "dewey.irods.host")
-              (get props "dewey.irods.port")
-              (get props "dewey.irods.user")
-              (get props "dewey.irods.password")
-              (get props "dewey.irods.home")
-              (get props "dewey.irods.zone")
-              (get props "dewey.irods.default-resource")))
+  []
+  (irods/init (cfg/irods-host)
+              (cfg/irods-port)
+              (cfg/irods-user)
+              (cfg/irods-pass)
+              (cfg/irods-home)
+              (cfg/irods-zone)
+              (cfg/irods-default-resource)))
 
 
 (defn- listen
-  [props irods-cfg es]
+  [irods-cfg es]
   (let [attached? (try
-                    (amq/attach-to-exchange (get props "dewey.amqp.host")
-                                            (Integer. (get props "dewey.amqp.port"))
-                                            (get props "dewey.amqp.user")
-                                            (get props "dewey.amqp.password")
-                                            (str "indexing." (get props "dewey.environment-name"))
-                                            (get props "dewey.amqp.exchange.name")
-                                            (Boolean. (get props "dewey.amqp.exchange.durable"))
-                                            (Boolean. (get props "dewey.amqp.exchange.auto-delete"))
-                                            (Integer. (get props "dewey.amqp.qos"))
+                    (amq/attach-to-exchange (cfg/amqp-host)
+                                            (cfg/amqp-port)
+                                            (cfg/amqp-user)
+                                            (cfg/amqp-pass)
+                                            (str "indexing." (cfg/environment-name))
+                                            (cfg/amqp-exchange)
+                                            (cfg/amqp-exchange-durable)
+                                            (cfg/amqp-exchange-autodelete)
+                                            (cfg/amqp-qos)
                                             (partial curation/consume-msg irods-cfg es)
                                             "data-object.#"
                                             "collection.#")
                     (log/info "Attached to the AMQP broker.")
                     true
                     (catch Exception e
-                      (log/debug (str e))
-                      (log/info "Failed to attach to the AMQP broker. Retrying...")
+                      (log/info e "Failed to attach to the AMQP broker. Retrying...")
```

Comment:
This is the only non-config-related change; logging the error this way means the stacktrace will get logged.